shared/settings/nav/index.desktop.js

```diff
@@ -24,7 +24,7 @@ function SettingsNav({badgeNumbers, selectedTab, onTabChange, onLogout}: Props)
   return (
     <Box style={styleNavBox}>
       <SettingsItem
-        text="Your Account"
+        text="Your account"
```

Comment:
This won't make a difference because the text is in all caps. And in general for titles, we might want all non-connective words capitalized.

Reply:
Fair enough. I defer to your judgment!